src/test/java/http/router/RouterTest.java

```diff
@@ -0,0 +1,89 @@
+package http.router;
+
+import http.handlers.request.*;
+import http.request.HttpRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RouterTest {
+
+    private HttpRequest httpRequestGet,
+                        httpRequestPost,
+                        httpRequestDelete,
+                        httpRequestOptions,
+                        httpRequestPut,
+                        httpRequestInvalid;
+
+    @BeforeEach
+    void setUp() {
+        httpRequestGet = new HttpRequest(rawGetRequest());
+        httpRequestPut = new HttpRequest(rawPutRequest());
+        httpRequestPost = new HttpRequest(rawPostRequest());
+        httpRequestDelete = new HttpRequest(rawDeleteRequest());
+        httpRequestOptions = new HttpRequest(rawOptionsRequest());
+        httpRequestInvalid = new HttpRequest(rawInvalidRequest());
+    }
+
+
+
+    @Test
+    void getResponse() {
```

Comment:
Test name seems a little vague.

Reply:
@td-extreme I'll rename it something more descriptive like "returns appropriate handler"